README.md

```diff
@@ -44,7 +46,9 @@ Upcoming enhancements are captured as druid-enhancement-proposal [a.k.a DEP] [he
 If you wish to contribute then please see our [guidelines](https://github.com/gardener/etcd-druid/blob/4e9971aba3c3880a4cb6583d05843eabb8ca1409/CONTRIBUTING.md).
 
 ## Feedback and Support
-We always look forward to active community engagement. Please report bugs or suggestions on how we can enhance `etcd-druid` on [Github Issues](https://github.com/gardener/etcd-druid/issues).
```

Comment:
Markdown best practices dictate that there is a newline between a heading and the following text. Adding it in my PR.